docs/ReleaseNotes-2.1.html

```diff
@@ -98,10 +98,8 @@ <h2><a name="platform">Platform Notes</a></h2>
     <hr>
     <h2><a name="install_setup_run">Installing, Setting Up, and Running the
         Jakarta Mail TCK</a></h2>
-    <p>Refer to the <cite><a href="html-usersguide/toc.html">Jakarta
-          mail TCK 2.1 User's Guide</a></cite> (or <a href="pdf-usersguide/Jakarta-Mail-TCK-Users-Guide.pdf"
-
-        title="Jakarta Mail TCK 2.1 User's Guide (PDF)">PDF</a>) for
+    <p>Refer to the <cite><a href="https://github.com/eclipse-ee4j/mail-tck/blob/master/QuickStart.txt">Jakarta
+          mail TCK 2.1 QuickStart.txt</a></cite> for
```

Comment:
There can be a pointer to up-to-date instructions but the file should also be part of the distribution zip.

Reply:
Done, finally it points to the local file.